solr/core/src/java/org/apache/solr/schema/ManagedIndexSchemaFactory.java

```diff
@@ -290,8 +301,9 @@ public ManagedIndexSchema create(
     return schema;
   }
 
-  private InputStream readSchemaLocally() {
+  private InputStream readSchemaLocally(String[] loadedResourceRef) {
```

Comment:
Couldn't we return a class or record or something else that you could pull the `InputStream` and `loadedResource` out of? This is a private method so no need to have the whole array stuff?

Reply:
👍 ... addressed this with 39c5620cd8f7ae96f98dde384b443ebe6d6ff318